### PR TITLE
[luci] Extract visitors to quantize activation

### PR DESCRIPTION
**Pull request by jinevening**
This extracts visitors to quantize activation from QuantizeWithMinMax.cpp.
- QuantizeActivation
- QuantizeConstInputActivation
- QuantizeSpecialActivation

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/8524

### CODE REVIEW

**Comment by jinevening** [comment]
I force-pushed just to update the commit message more properly.

File: compiler/luci/pass/src/QuantizeActivation.cpp [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+{
+  assert(node->dtype() == loco::DataType::FLOAT32);
+
+  float min = std::numeric_limits<float>::max();
+  float max = std::numeric_limits<float>::lowest();
+  for (uint32_t i = 0; i < node->size<loco::DataType::FLOAT32>(); i++)
+  {
+    auto data = node->at<loco::DataType::FLOAT32>(i);
+    min = data < min ? data : min;
+    max = data > max ? data : max;
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizeActivation.cpp:40]: (optional) maybe use `std::min()`, `std::max()` ?

**Review by seanshpark — APPROVED** [review]
LGTM

File: compiler/luci/pass/src/QuantizeActivation.cpp (hunk 1 above)
Comment (seanshpark) [reply compiler/luci/pass/src/QuantizeActivation.cpp]: ah, maybe need to include `<algorithm>` too :)

**Review by seanshpark — APPROVED** [review]
LGTM thank you

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**